data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/config/GetSpec.java

```diff
@@ -53,12 +68,18 @@ public GetSpec(final DataProvider dataProvider, final JsonProvider jsonProvider)
         @OpenApiResponse(status = RES_INTERNAL_ERROR)
       })
   @Override
-  public void handle(final Context ctx) throws Exception {
+  public void handle(@NotNull final Context ctx) throws Exception {
+    adapt(ctx);
+  }
+
+  @Override
+  public void handleRequest(RestApiRequest request) throws JsonProcessingException {
     try {
-      ctx.json(jsonProvider.objectToJSON(configProvider.getConfig()));
```

Comment:
does anything still call that `getConfig` other than tests?

Reply:
Three test files call `getConfig`

Reply:
<img width="832" alt="Screen Shot 2022-04-07 at 1 08 55 PM" src="https://user-images.githubusercontent.com/45641759/162100016-7604f661-3062-4aef-b2d3-cdd50c21bd3b.png">